packages/lib/src/components/Badge/TaskStatusBadge/index.tsx

```diff
@@ -2,10 +2,10 @@ import { css } from "@emotion/react";
 import styled from "@emotion/styled";
 import Badge, { Variant } from "@leafygreen-ui/badge";
 import { palette } from "@leafygreen-ui/palette";
-import { TaskStatus } from "@evg-ui/lib/types/task";
-import { taskStatusToCopy } from "constants/task";
+import { taskStatusToCopy } from "../../../constants/task";
+import { TaskStatus } from "../../../types/task";
```

Comment:
For some reason, I can't get these absolute imports working from within Parsley. Does anyone have any suggestions? I assume something is wrong with tsconfig-resolve-paths but I'm not sure.

Reply:
did you mean from within `lib`? I think in this case you can just do

`import { taskStatusToCopy } from "constants/task";`

Reply:
Importing within lib passes but when I do that I get this [error](https://parsley.mongodb.com/evergreen/evergreen_ui_parsley_compile_patch_7a69ecadec70775e06d1dd8b01793117ad0824ff_66f2ff388ab67400071e783b_24_09_24_18_04_41/0/task?bookmarks=0%2C191&selectedLineRange=L110-L111&shareLine=110) in the Parsley tests

Reply:
I filed https://jira.mongodb.org/browse/DEVPROD-11621